raspihats: unmet dependency fix

szogi opened the pull request:
Raspihats platform update, upstream fixed enum34 requirements, added smbus dependency

**Related issue (if applicable):** fixes #9547 

**Pull request in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io) with documentation (if applicable):** home-assistant/home-assistant.github.io#3483

## Checklist:

If user exposed functionality or configuration variables are added/changed:
  - [X] Documentation added/updated in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io)

If the code communicates with devices, web services, or third-party tools:
  - [X] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**
  - [X] New dependencies have been added to the `REQUIREMENTS` variable ([example][ex-requir]).
  - [X] New dependencies are only imported inside functions that use them ([example][ex-import]).
  - [X] New dependencies have been added to `requirements_all.txt` by running `script/gen_requirements_all.py`.

[ex-requir]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L14
[ex-import]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L54

szogi committed:
raspihats: update to 2.2.3 (deps fix)

Raspihats platform update, upstream fixed enum34 requirements, added smbus dependency
Fixes #9547

szogi committed:
raspihats: update to 2.2.3, smbus-cffi dependency

Raspihats platform update, upstream fixed enum34 requirements, added smbus dependency
Fixes #9547

szogi committed:
raspihats: update to 2.2.3

szogi committed:
raspihats: update to 2.2.3, smbus-cffi dependency

szogi committed:
raspihats: update to 2.2.3, smbus-cffi dependency

szogi commented:
huh, it was tricky 🔨  :)

Pistahh commented on homeassistant/components/raspihats.py:
```diff
@@ -12,7 +12,8 @@
 )
 
-REQUIREMENTS = ['raspihats==2.2.1']
+REQUIREMENTS = ['raspihats==2.2.3'
```
Missing comma. This one just concats the strings.

    >>> REQUIREMENTS = ['raspihats==2.2.3'
    ...                 'smbus-cffi==0.5.1']
    >>> REQUIREMENTS
    ['raspihats==2.2.3smbus-cffi==0.5.1']
    >>>

szogi replied:
ah, ty! :)

szogi committed:
raspihats: update to 2.2.3 (deps fix)

Raspihats platform update, upstream fixed enum34 requirements, added smbus dependency
Fixes #9547

szogi committed:
raspihats: update to 2.2.3, smbus-cffi dependency

balloob commented:
Thanks!
